### ISSUE TITLE
Size of windows exe exprloded between 2.1 and 3.0

### ISSUE THREAD

**Issue by berhoel** [pyinstaller/pyinstaller#1814]
I have a project that I want to convert to a exe file. I use a setup unsing "--onedir". With the switch tp PyInstaller 3.0 we observed an increase of the dist directory from 56MB to 754MB, and if I inspect the dist directory I find lots of duplicated and unused information. The whole "%PYTHONPATH%" directory is copied besides those projects required.

**Comment by htgoebel** [comment]
For Python 3 and for unicode-support, Pyinstaller needs to in include *a lot* of stuff. including the whole `encoding` and `codecs` package.

Beside this can you please elaborate what exactly the "duplicated and unused information" is?

**Comment by berhoel** [comment]
I used a virtualenv to install all prequisites of my project including PyInstaller. Somehow I managed to include the whole virtual environment in the 'dist' directory. After beeing unable to reproduce my problem with a small, managable example, I fiddeled with my hooks and managed to fix this.

I use Inno Setup 5 to generate an installer from the 'dist' directory. With 2.1 the resulting installer is 17MB, using 3.1.1it's 22MB after fiddeling with the hooks. For our purpose 17MB ist already a lot, so, for the time beeing, we probably will stick with 2.1.

It seems my original problem is mostly a user error/problem so I close this issue.
